src/pageTypes/components/PageTypeListPage/PageTypeListPage.tsx

```diff
@@ -36,38 +41,71 @@ const PageTypeListPage: React.FC<PageTypeListPageProps> = ({
   onTabChange,
   onTabDelete,
   onTabSave,
+  onTabUpdate,
+  hasPresetsChanged,
   tabs,
   ...listProps
 }) => {
   const intl = useIntl();
+  const navigate = useNavigator();
+  const [isFilterPresetOpen, setFilterPresetOpen] = useState(false);
 
   return (
     <ListPageLayout>
-      <TopNav href={configurationMenuUrl} title={intl.formatMessage(sectionNames.pageTypes)}>
-        <Button variant="primary" href={pageTypeAddUrl} data-test-id="create-page-type">
+      <TopNav
+        isAlignToRight={false}
+        withoutBorder
+        href={configurationMenuUrl}
+        title={intl.formatMessage(sectionNames.pageTypes)}
+      >
+        <Box __flex={1} display="flex" justifyContent="space-between" alignItems="center">
+          <Box display="flex">
+            <Box marginX={3} display="flex" alignItems="center">
+              <ChevronRightIcon />
+            </Box>
+
+            <FilterPresetsSelect
+              presetsChanged={hasPresetsChanged()}
+              onSelect={onTabChange}
+              onRemove={onTabDelete}
+              onUpdate={onTabUpdate}
+              savedPresets={tabs}
+              activePreset={currentTab}
+              onSelectAll={onAll}
+              onSave={onTabSave}
+              isOpen={isFilterPresetOpen}
+              onOpenChange={setFilterPresetOpen}
+              selectAllLabel={intl.formatMessage({
+                id: "ER0uUn",
+                defaultMessage: "All page types",
+                description: "select all page types preset label",
+              })}
+            />
+          </Box>
+        </Box>
+
+        <Button
+          onClick={() => navigate(pageTypeAddUrl)}
+          variant="primary"
+          data-test-id="create-page-type"
+        >
           <FormattedMessage id="6JlXeD" defaultMessage="Create page type" description="button" />
         </Button>
       </TopNav>
-      <DashboardCard>
-        <SearchBar
-          allTabLabel={intl.formatMessage({
-            id: "oVDZUb",
-            defaultMessage: "All Page Types",
-            description: "tab name",
-          })}
-          currentTab={currentTab}
-          initialSearch={initialSearch}
-          searchPlaceholder={intl.formatMessage({
-            id: "umsU70",
-            defaultMessage: "Search Page Type",
-          })}
-          tabs={tabs}
-          onAll={onAll}
-          onSearchChange={onSearchChange}
-          onTabChange={onTabChange}
-          onTabDelete={onTabDelete}
-          onTabSave={onTabSave}
-        />
+      <DashboardCard gap={0}>
+        <Box paddingX={6} marginBottom={2}>
+          <Box __width="320px">
+            <SearchInput
+              initialSearch={initialSearch}
+              placeholder={intl.formatMessage({
+                id: "umsU70",
+                defaultMessage: "Search Page Type",
```

Comment:
Suggestion: A teeny tiny nit pick but in line 92 and 80 we don't use capital letters for "page type" but here we do. Maybe we should just keep "Search" here or "Type to search"? Do we have any pattern for this?

Reply:
We have common pattern across the other views "Search <entity name>" eg. "Search attributes" etc. so we should follow same here "Search page types"